std/utf.d

```diff
@@ -937,6 +1031,12 @@ unittest
 
 /* =================== Decode ======================= */
 
+/**
+    $(D yes) means replace invalid UTF encodings with useReplacementDchar,
+    $(D no) means throw an exception when invalid UTF encodings are decoded.
+ */
+alias UseReplacementDchar = Flag!"useReplacementDchar";
```

Comment:
@klickverbot yah, the idea here is that introducing alias here is the opposite of friendliness - makes folks go look for the alias instead of knowing what to do. Seeing `Flags!"xyz"` makes it clear that `Yes.xyz` or `No.xyz` may be passed.